packages/esm-ward-app/src/ward-patient-card/bento-elements/ward-patient-age.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { type WardPatientCardBentoElement } from '../../types';
+
+const WardPatientAge: WardPatientCardBentoElement = ({ patient }) => {
+  const { t } = useTranslation();
+  return (
+    <div>
+      {t('yearsOld', '{{age}} yrs', {
+        age: patient?.person?.age,
+      })}
```

Comment:
I'd probably suggest using esm-framework's `age()` function here. That handles years, months, weeks, and days (may need some extension for hours).

Reply:
+1

Reply:
The backend isn't returning the birthdate either. I'll add a TODO.

Reply:
There is [this PR](https://github.com/Bahmni/openmrs-module-appointments/pull/148). May be we push for its review

Reply:
Ah... I hadn't looked into the backend, but it looks like the fields returned from patient [are hard-coded](https://github.com/Bahmni/openmrs-module-bedmanagement/blob/2058e237664331f90a2172a8529920e9a53cee00/omod/src/main/java/org/openmrs/module/bedmanagement/rest/resource/AdmissionLocationResource.java#L190). Here's a ticket to adjust that: https://openmrs.atlassian.net/browse/BED-10.